src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Ball();
```

Comment:
The variable 'ball' should be of type 'Ball' instead of 'Lottery' if you are not planning to assign other implementations of the 'Lottery' interface to it.